mlcp/src/main/java/com/marklogic/contentpump/ContentPump.java

```diff
@@ -152,10 +152,6 @@ public static int runCommand(String[] args) throws IOException {
                 MODE + ": " + mode);
             return 1;
         }
-        if (MODE_DISTRIBUTED.equalsIgnoreCase(mode)) {
-            LOG.warn("MLCP distributed mode will be disabled in the next " +
```

Comment:
Can we proactively put a link to the docs for more information?

Reply:
A link in the error message or a link in the documentation/release note?

Reply:
Some sort of look-up in the log message, like an error code or a URL, so it’s to find the the doc (or more likely via Google).

Reply:
Okay, I will add a url to our mlcp user guide which has more details on the deprecation.